src/main/flight/gps_rescue.c

```diff
@@ -380,7 +380,7 @@ static void performSanityChecks(void)
     static float prevTargetAltitudeCm = 0.0f; // to calculate ascent or descent target change
     static float previousDistanceToHomeCm = 0.0f; // to check that we are returning
     static int8_t secondsLowSats = 0; // Minimum sat detection
-    static int8_t secondsDoingNothing = 0; // Limit on doing nothing
+    static int8_t secondsDoingNothing; // Limit on doing nothing
```

Comment:
This was needed for RESCUE_DO_NOTHING state to finish after 20ms.

Reply:
I'd rather see 10s as maximum here. You really have steel nerves @ctzsnooze testing this :)

Reply:
Why wouldn't we initialise it to zero?

Reply:
Tested, While there it didn't disarm after 20 seconds in `DO_NOTHING` mode (the countdown does not work - as it's reset on each iteration of the sanity check).

🤔 Did not touch any other values, The reset should take place in INITIALIZE mode only?